docker/Dockerfile

```diff
@@ -222,6 +223,13 @@ RUN git clone --recursive https://github.com/pytorch/serve.git \
 COPY --from=compile-image /home/venv /home/venv
 ENV PATH="/home/venv/bin:$PATH"
 WORKDIR "serve"
+RUN \
+    if [ "$BUILD_CPP" = "true" ]; then \
+        python ./ts_scripts/install_dependencies.py --environment=dev --cpp --force \
+        && cd cpp \
+        && ./build.sh \
```

Comment:
this will make the docker image too big. you only need make sure build.sh can work in the docker image.

Reply:
Makes sense, will remove this RUN command.